docs/index.md

```diff
@@ -5,46 +5,33 @@ hide:
-can easily drag and drop elements, customize layouts, and integrate features effortlessly.
+The capability to:
 
-Taipy is a versatile tool that enables both individuals and businesses to create applications 
-without needing advanced coding skills. One of its key benefits is how it speeds up development for 
-both the front-end and back-end aspects, covering everything from initial prototypes to fully 
-scalable, ready-to-use applications.
+- Register pipeline single execution. 
+- Perform what-if analysis (comparing different executions/runs). 
+- Explore your past executions, comparing them.
+
+Is provided by Taipy’s “Scenario Management”, one of Taipy’s central concepts.
 
 **Front-End Functionalities:**
 
-- Creating a user interface requires a solid grasp of Python programming fundamentals.
-- Taipy is purposefully engineered to prioritize user-friendliness, resulting in a straightforward 
-  and intuitive process of user interface creation.
-- No prior knowledge of web design is necessary, and it eradicates the need for any prerequisites 
-  related to CSS and HTML.
-- Leveraging augmented Markdown syntax, Taipy GUI aids users in seamlessly generating their desired
-  web pages.
+- Taipy provides all the GUI components needed to build interactive front-ends.
+- No prior knowledge of web design is necessary, and any prerequisites related to CSS and HTML.
+- Leveraging augmented Markdown syntax, Taipy GUI aids users in seamlessly generating their desired web pages.
+- Taipy’s ability to really customize your GUI in terms of layout, graphical events, Style/CSS, etc is one of Taipy’s strongest points.
 
 **Back-End Functionalities:**
 
-- Taipy provides extensive support for establishing resilient pipelines capable of managing diverse
-  scenarios.
+- Taipy provides extensive support for orchestrating resilient pipelines capable of managing diverse situations.
 - The process of modeling Directed Acyclic Graphs (DAGs) is simplified through Taipy Studio.
-- Enhancing the overall performance of Taipy applications, data caching, parallel executions of 
-  tasks, data source scoping are all integrated.
+- Enhancing the overall performance of Taipy applications, task caching, parallel executions of tasks, data source scoping are all integrated.
 - Providing a registry of pipeline executions.
-- Taipy equips users with the ability to monitor and assess the performance of their applications 
-  using the KPI tracking tool.
-- Taipy offers built-in graphical explorers for visualizing, editing, and monitoring your ‘executed’
-  pipelines and their associated data.
+- Taipy equips users with the ability to monitor and assess the performance of their applications using the KPI tracking tool.
+- Taipy offers built-in graphical explorers for visualizing, editing, and monitoring your past ‘executed’ pipelines and their associated data.
 
 ## Overview of Taipy
```

Comment:
In the end, I believe this illustration should not be on the homepage. Eventually on the user manual index page to get an overview of the way it works. But surely not on the Doc Homepage.

Reply:
Then I would move it to Getting Started, as the diagram is important to understand Taipy. In the User Manual, it will be lost a bit, or too late to review after they went via the Getting started.